hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/client/rpc/TestOzoneRpcClientWithRatis.java

```diff
@@ -103,7 +106,7 @@ public void testGetKeyAndFileWithNetworkTopology() throws IOException {
     builder.setVolumeName(volumeName).setBucketName(bucketName)
         .setKeyName(keyName).setRefreshPipeline(true);
 
-    // read key with topology aware read enabled(default)
+    // read key with topology aware read enabled
     try {
       OzoneInputStream is = bucket.readKey(keyName);
```

Comment:
Should we make these OzoneInputStream closed after they are done.

Reply:
Yes. They should be closed.